mlrun/model.py

```diff
@@ -774,20 +774,22 @@ def validate_notification_params(self):
 
         notification_class.validate_params(secret_params | params)
 
-    def fill_params_from_secret_params(self):
+    def fill_secret_params_from_project_secret(self):
```

Comment:
rename to `unmask_notification_params_secret` it should match the server side function as it does a similar operation